pkgs/by-name/pa/passhole/package.nix

```diff
@@ -0,0 +1,68 @@
+{
+  lib,
+  fetchPypi,
+  fetchurl,
+  python3,
+  stdenv,
+}:
+let
+  pname = "passhole";
+  version = "1.10.0";
+
+  pykeepass-cache = python3.pkgs.buildPythonPackage {
+    pname = "pykeepass-cache";
+    version = "2.0.3";
+    src = fetchurl {
```

Comment:
`passhole` depends on `pykeepass-cache`, which isn't available in nixpkgs python packages, so I had to build it from Pypi (https://pypi.org/project/pykeepass-cache/). I was fetching the tarball using fetchurl earlier because `fetchPypi` didn't work for me for some reason (might have been due to some hash troubles); however I managed to get it working with `fetchPypi` in my most recent commit (02ef20b162a1940641bc68fe8878eb5517691e62).

Reply:
Please use fetchPypi here (maybe it got lost in a rebase)